src/Stripe/Services/Customers/StripeCustomerCreateOptions.cs

```diff
@@ -28,6 +28,9 @@ public class StripeCustomerCreateOptions
         [JsonProperty("plan")]
         public string PlanId { get; set; }
 
+        [JsonProperty("tax_percent")]
+        public decimal? TaxPercent { get; set; }
```

Comment:
I don't see this property in the [Stripe API Reference](https://stripe.com/docs/api#create_customer). Where did you find this?